docs/source/support/faq.rst

```diff
@@ -46,66 +73,85 @@ Q: Who is the Staker in this narrative? Is it Alice or Ursula?
 The Staker can be thought of as a fiduciary administrator that holds NU and collects rewards.
 Typically, but not always, Ursula and the Staker are the same party. Recall that Ursula is only “valid” (i.e. will be selected for work and able to earn inflation rewards) if she’s bonded to a Staker.
 
-Q: What kind of token is NU?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-NU is an implementation of the ERC20 standard deployed onto the Ethereum blockchain.
-
-Q: Why have the Staker and Ursula been split?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Q: Does Alice or Bob need NU to use the network? Who pays Ursula? How is it done?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We split them so that the Staker can hold NU offline in a hardware wallet.
+Neither Alice nor Bob need NU to use the network - Stakers require NU to run Ursulas on the network. However, Alice pays Ursulas for re-encryptions via an ETH deposit to the PolicyManager contract at the point of granting access to Bob. Payment models are an area of active research for the NuCypher protocol, including the prospect of Bob independently paying in ETH for all or some component of the work performed by Ursula.
 
-Q: What currency does Alice use to pay for re-encryptions?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Q: How much does Alice pay?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Alice pays for re-encryptions in Ether.
+Alice provides an upfront deposit that covers the entirety of the sharing policy's duration (in periods), for all the Ursulas required to service the policy. Currently, Ursulas may choose a discretionary payment rate per period, which Alices discover on an Ursula-by-Ursula basis by querying public network contracts. In the early stages of the network, both Alice and Ursula must settle on a rate which falls within a fixed, universal range, enforced by the PolicyManager contract.
 
 Q: What currency does Ursula stake in (assuming Ursula is also the Staker)?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Ursula stakes in NU, collects policy rewards in ETH, and inflation rewards in NU.
-
-Q: What are the two streams of income Ursula can receive?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Inflation Rewards (NU) and Policy Rewards (ETH). We will soon refer to Policy Rewards as “Fees” to avoid confusion.
+Ursula stakes in NU, receives policy rewards in ETH, and receives inflation rewards in NU.
 
 Q: Why do you have a mix of NU and ETH?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 It’s much more convenient for Alice to simply carry ETH. If she has to acquire NU also, it sets a much higher barrier to entry.
```

Comment:
```suggestion
It’s much more convenient for Alice to simply carry ETH. If she has to acquire NU also, it sets a much higher barrier to entry.
Additionally, since NU is an ERC20 token, using it for payment instead of ETH offers no advantages.
```